src/main/java/com/epam/lowcost/controller/PlaneController.java

```diff
@@ -0,0 +1,59 @@
+package com.epam.lowcost.controller;
+
+import com.epam.lowcost.model.Plane;
+import com.epam.lowcost.service.PlaneService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.Map;
+
+@Controller
+@RequestMapping(value = "/plane")
+public class PlaneController {
+
+    @Autowired
+    PlaneService planeService;
+
+    @GetMapping(value = "/all")
+    public String listAllUsers(Model model) {
```

Comment:
list - it is noun. Method - need to be a verb

Reply:
Changed method name